mods/settings.md

```diff
@@ -579,6 +579,55 @@ $execute {
 }
 ```
 
+Incase you want to store a more complex value (e.g. a string or a custom class/struct), you'll have to implement all the required operators. This is done as follows:

```

Comment:
Kinda janky language, I would write this as `If you want to have a setting with a more complex type (e.g. a string or a custom class/struct), the minimum requirement is that the class is JSON-serializable and copy constructible. For example:`